scripts/version

```diff
@@ -78,4 +78,4 @@ echo "APP_VERSION: $APP_VERSION"
 echo "SUFFIX: $SUFFIX"
 echo "REPO: $REPO"
 echo "TAG: $TAG"
-echo "AGENT_TAG: $AGENT_TAG"
+echo "AGENT_TAG: $AGENT_TAG"
```

Comment:
This is going to annoy the next person working on this file

Reply:
Fixed